utils/docker/Dockerfile.centos:7

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2016-2019 Intel Corporation
```

Comment:
Filename not portable to all systems used for development.

Reply:
This file wasn't actually even being used.  Removed.